schema/operation.json

```diff
@@ -3,37 +3,118 @@
     "$id": "http://standards.ncats.io/operation.json",
     "anyOf": [
         {
-            "$ref": "#/definitions/OperationOverlay"
+            "$ref": "#/definitions/OperationCompleteResults"
```

Comment:
@patrickkwang It appears the operations are still not sorting themselves. No big deal, but I know you wanted to avoid this.